### PR TITLE
why don't you give me a slack

### PR DESCRIPTION
**Pull request by krastegar**
Be nice to my code

### CODE REVIEW

**Review by Zack1803 — APPROVED** [review]
Looks good to me.

File: src/Week2_HW.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            }
+        ]
+    )
+    return report
+
+
+def main():
+
+    # loading iris dataset from sklearn
+    iris = load_iris()
```
Comment (Zack1803) [inline src/Week2_HW.py]: Are we allowed to use load_iris()? I think it would be better to stick with iris.data file.

Comment (dafrenchyman) [reply src/Week2_HW.py]: That is some very sound advice!

**Comment by nmohieddin** [comment]
looks great!

**Pull request merged**